src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -407,6 +418,10 @@ private void createMesosTask(Offer offer, Request request) {
     LOGGER.info("Launching task " + taskId.getValue() + " with URI " +
                 joinPaths(jenkinsMaster, SLAVE_JAR_URI_SUFFIX));
 
+    if (isExistingTask(offer, taskId)) {
```

Comment:
If I remember correctly it's when you receive an multiple offers for the same task id at different times

On June 21, 2015 9:57:39 PM CDT, Vinod Kone <notifications@github.com> wrote:
>> @@ -407,6 +418,10 @@ private void createMesosTask(Offer offer,
>Request request) {
>>      LOGGER.info("Launching task " + taskId.getValue() + " with URI "
>+
>>                  joinPaths(jenkinsMaster, SLAVE_JAR_URI_SUFFIX));
>>  
>> +    if (isExistingTask(offer, taskId)) {
>
>Do you know, when is this possible to get into this state? i.e., a
>createMesosTask() is called with an existing task?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/jenkinsci/mesos-plugin/pull/124/files#r32902777

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.